### PR TITLE
update datadog drains

### PR DESCRIPTION
**Pull request by KannarFr**

### CODE REVIEW

**Comment by KannarFr** [comment]
@hsablonniere can you ping here when new console version will be released?

File: commons/clever-tools/lifecycle.md [hunk 1]
```diff
@@ -37,11 +37,9 @@ Where `DRAIN-TYPE` is one of:
  - `TCPSyslog`: for TCP syslog endpoint;
  - `UDPSyslog`: for UDP syslog endpoint;
  - `HTTP`: for TCP syslog endpoint (note that this endpoint has optional username/password parameters as HTTP Basic Authentication);
- - `ElasticSearch`: for ElasticSearch endpoint (note that this endpoint requires username/password parameters as HTTP Basic Authentication);
- - `Datadog`: for Datadog endpoint (note that this endpoint requires --api-key parameter as Datadog authentication).
+ - `ElasticSearch`: for ElasticSearch endpoint (note that this endpoint requires username/password parameters as HTTP Basic Authentication).
```
Comment (urcadox) [inline commons/clever-tools/lifecycle.md]: Why did you s/;/./?

Comment (urcadox) [reply commons/clever-tools/lifecycle.md]: It's not the last element

File: commons/clever-tools/lifecycle.md [hunk 2]
```diff
@@ -37,11 +37,9 @@ Where `DRAIN-TYPE` is one of:
...
+ - `DatadogHTTP`: for Datadog endpoint (note that this endpoint need your Datadog API Key).
```
Comment (urcadox) [inline commons/clever-tools/lifecycle.md]: needs*

File: commons/clever-tools/lifecycle.md [hunk 3]
```diff
@@ -50,6 +48,18 @@ And remove them if needed
 
     clever drain remove [--alias <alias>] <DRAIN-ID>
 
+#### ElasticSearch logs drains
+
+ElasticSearch drains use the Elastic bulk API. To match this endpoint, specify `/_bulk` at the end of your ElasticSearch endpoint.
+
+#### Datadog logs drains
+
+Datadog has two zones, EU and COM. An account on one zone is not available on the other, make user to target the good EU or COM intake endpoint.
```
Comment (urcadox) [inline commons/clever-tools/lifecycle.md]: ... s/user/sure/?

File: commons/clever-tools/lifecycle.md [hunk 4]
```diff
@@ -50,6 +48,18 @@ And remove them if needed
...
+
+To create [Datadog](https://docs.datadoghq.com/api/?lang=python#send-logs-over-http) drain, you just need to use:
```
Comment (urcadox) [inline commons/clever-tools/lifecycle.md]: s/create/create a/

**Pull request merged**